Ochowo/iReporter
# #162465300 Create an  intervention

**What does this PR do?**
This pull request creates an endpoint for creating an intervention

**Description of Task to be completed?**
- Creates the endpoint to POST an intervention
 `POST/api/v1/interventions`

**How should this be manually tested?**
After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have
`nodejs` and `express` installed. Afterwards, run `npm start:dev` to initiate the app and `npm test` for test
- test the endpoint using postman with localhost:3000 as the host
- send empty inputs to check validation

**What are the relevant pivotal tracker stories?**
- [#162465300](https://www.pivotaltracker.com/story/show/162465300)

**Any background context you want to add?**
- N/A

**Screenshots**
- N/A
